### PR TITLE
add no-sequential-design sampling

### PR DESCRIPTION
**Pull request by XingLLiu**
1.updated integrationMain.R to incorporate no-sequential-design sampling.
-To run without sequential design:
`Rscript integrationMain.R dim num_iter which_genz 0`
-To run with sequential design, either replace 0 by 1 in the above command or simply omit the last input

2.fixed directory bug in integrationMain.R

3.changed the output figure name format. Now will produce something like:
-copeak3Dim, for sequential design
-copeak3DimNoSequential, for no sequential design

### CODE REVIEW

File: src/mixedGenzMain.R [hunk 1]
```diff
@@ -86,14 +86,14 @@ results <- data.frame(
...
-        # "runtimeGP" = rep(GPTime, num_iterations)
+        "runtimeGP" = rep(GPTime, num_iterations)
 )
 # write.csv(results, file = "../results/genz/%s/results%sdim%s.csv" %--% c(whichGenz, whichGenz, dim),row.names=FALSE)
 
 print("Begin Plots")
 
 # 1. Open jpeg file
-jpeg("../Figures/mixedGenz.jpg" %--% c(whichGenz, genzFunctionName, dim), width = 700, height = 583)
+jpeg("../figures/mixedGenz.jpg" %--% c(whichGenz, genzFunctionName, dim), width = 700, height = 583)
```
Comment (harrisonzhu508) [inline src/mixedGenzMain.R]: Should be Figures

File: integrationMain.R [hunk 2]
```diff
@@ -149,6 +152,6 @@ legend("topleft", legend=c("MC Integration", "BART BQ", "GP BQ"),
 # 3. Close the file
 dev.off()
 
-print("Please check {ROOT}/report/Figures for plots")
+print("Please check {ROOT}/figures/%s for plots" %--% figName)
```
Comment (harrisonzhu508) [inline integrationMain.R]: Figures

**Comment by harrisonzhu508** [comment]
The `test_step_func` should be in the root directory (behind src)

**Comment by XingLLiu** [comment]
All fixed.

**Comment by harrisonzhu508** [comment]
Cheers. Although test_step_func.R should be moved to ../src?

**Comment by harrisonzhu508** [comment]
Other please do merge the pull request

**Comment by XingLLiu** [comment]
> Cheers. Although test_step_func.R should be moved to ../src?

Moved to root dir.

**Pull request merged**